Assets/MixedRealityToolkit/_Core/Definitions/Utilities/ExperienceScale.cs

```diff
@@ -22,7 +22,7 @@ public enum ExperienceScale
         /// </summary>
         Standing,
         /// <summary>
-        /// An experience designed to support movement thoughtout a room. The coordinate system origin is at floor level.
+        /// An experience designed to support movement thought out a room. The coordinate system origin is at floor level.
```

Comment:
throughout

Reply:
huh, I'm getting a type warning in my spell checker with `throughout`

Reply:
through out is fine too if that plays nicer with spell check, just not "thought out"

Reply:
I get it now. Sorry 😅

Reply:
Haha all good!